docs/DeployConfigurations.rst

```diff
@@ -9,19 +9,27 @@ Currently, only one kind of application deployment is supported:
 Docker Deploy Daemon
 --------------------
 
-Provides the deployment of applications as Docker containers.
+Orchestrates the deployment of applications as Docker containers.
 
-There are several Docker Daemon Deploy configurations:
+.. warning::
+
+	The administrator is responsible for the security actions taken to configure the Docker daemon connected to Nextcloud.
+	For example, look at the Docker Socket Proxy protected by strict rules without root access (`like in AIO <#nextcloud-in-docker-aio-all-in-one>`_)
+
+There are several Docker Daemon Deploy configurations (example schemes):
 
 	* Nextcloud and Docker on the **same host** (via socket or port)
```

Comment:
as mentioned in the issue, should we only recommend rootless docker when not having a docker-socket-proxy in between?

Reply:
I think it's worth to display current configurations in comparison with our suggested and secure way for different possible option for local and remote daemons. So it's more clear why it's better to use docker-socket-proxy.